warning: `qSort()` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:488:10: warning: ‘void qSort(Container&) [with Container = QStringList]’ is deprecated: Use std::sort [-Wdeprecated-declarations]
  488 |     qSort(category_list);
      |     ~~~~~^~~~~~~~~~~~~~~
In file included from /usr/include/qt/QtCore/qlist.h:43,
                 from /usr/include/qt/QtCore/qobject.h:49,
                 from /usr/include/qt/QtWidgets/qwidget.h:45,
                 from /usr/include/qt/QtWidgets/qframe.h:44,
                 from /usr/include/qt/QtWidgets/qabstractscrollarea.h:44,
                 from /usr/include/qt/QtWidgets/qmdiarea.h:44,
                 from /usr/include/qt/QtWidgets/QMdiArea:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/EnhancedWorkspace.h:40,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPASBase.h:42,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:35:
/usr/include/qt/QtCore/qalgorithms.h:188:46: note: declared here
  188 | QT_DEPRECATED_X("Use std::sort") inline void qSort(Container &c)
      |                                              ^~~~~
```

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -485,7 +485,7 @@ namespace OpenMS
       category_set << String(it->second.category).toQString();
     }
     QStringList category_list = category_set.toList();
-    std::sort(category_list.begin(), category_list.end(), std::less<QString>());
+    std::sort(category_list.begin(), category_list.end());
     std::map<QString, QTreeWidgetItem*> category_map;
 
     foreach(const QString &category, category_list)

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -485,7 +485,7 @@ namespace OpenMS
       category_set << String(it->second.category).toQString();
     }
     QStringList category_list = category_set.toList();
-    qSort(category_list);
+    std::sort(category_list.begin(), category_list.end(), std::less<QString>());
     std::map<QString, QTreeWidgetItem*> category_map;
 
     foreach(const QString &category, category_list)

